bin/ssh_scan_api

```diff
@@ -22,6 +22,18 @@ opt_parser = OptionParser.new do |opts|
     exit
   end
 
+  opts.on("--https", "Use HTTPS") do
+    options[:https] = true
+  end
```

Comment:
I tried running the following and it didn't serve a self-signed cert by default....

./bin/ssh_scan_api --https